Uplift/Controllers/GymDetailViewController.swift

```diff
@@ -150,29 +150,13 @@ extension GymDetailViewController: UICollectionViewDataSource, UICollectionViewD
 
         switch itemType {
         case .hours:
-            let baseHeight = CGFloat(GymDetailHoursCell.Constants.hoursTitleLabelTopPadding +
-                GymDetailHoursCell.Constants.hoursTitleLabelHeight +
-                GymDetailHoursCell.Constants.hoursTableViewTopPadding +
-                GymDetailHoursCell.Constants.dividerTopPadding +
-                GymDetailHoursCell.Constants.dividerHeight)
-            let height = gymDetail.hoursDataIsDropped
-                ? baseHeight + CGFloat(GymDetailHoursCell.Constants.hoursTableViewDroppedHeight)
-                : baseHeight + CGFloat(GymDetailHoursCell.Constants.hoursTableViewHeight)
-            return CGSize(width: width, height: height)
+            return CGSize(width: width, height: hoursHeight())
```

Comment:
can we call these functions `getHoursHeight(), getBusyTimesHeight()`, etc. prefer these names rather than `hoursHeight` b/c that sounds like its a variable